mod/metrics/set/type/metric/creation.rb

```diff
@@ -6,6 +6,25 @@ def create_value_options options
   Card.create! create_args
 end
 
+event :ensure_designer, :prepare_to_store, on: :save, changed: :name do
+  return if valid_designer?
+  if (card = Card[metric_designer])
+    errors.add :metric_designer, "invalid type #{card.type_name}"
+  else
+    attach_subcard metric_designer, type_id: ResearchGroupID
+  end
+end
+
+event :ensure_title, :prepare_to_store, on: :save, changed: :name do
+  return if Card.fetch_type_id(metric_title) == MetricTitleID
+  attach_subcard metric_title, type_id: MetricTitleID
```

Comment:
no, that's just there so that it's more likely that new data is created correctly but I didn't want to deal with new issues (like failing imports because of existing non-metric title cards)